integration_tests/e2e/signIn.cy.ts

```diff
@@ -53,17 +53,17 @@ context('Sign In', () => {
     Page.verifyOnPage(IndexPage)
     cy.task('stubVerifyToken', false)
 
-    // can't do a visit here as cypress requires only one domain
-    cy.request('/').its('body').should('contain', 'Sign in')
+    cy.visit('/')
+    Page.verifyOnPage(AuthSignInPage)
```

Comment:
not sure what has changed, but looks like we can now do a visit and verify for other sites